src/main/java/com/google/sps/servlets/PostQuestionServlet.java

```diff
@@ -0,0 +1,123 @@
+      Connection connection = DriverManager
+          .getConnection(Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
+      insertNewQuestion(connection, title, body, asker_id);
+      insertNewFollower(connection, asker_id);
+    } 
+    catch (SQLException exception) {
+      // If the connection or the query don't go through, we get the log of what happened.
+      Logger logger = Logger.getLogger(PostQuestionServlet.class.getName());
+      logger.log(Level.SEVERE, exception.getMessage(), exception);
+    }
+    response.sendRedirect("/");
+  }
+
+  /** 
+   * Inserts a question into the database.
+   */
+  private void insertNewQuestion(Connection connection, String title, String body, int asker_id) {
+    try {
+      String insertQuestionQuery = "INSERT INTO Question(title, body, asker_id, date_time) "
+          + "VALUES (?,?,?,NOW())";
+      PreparedStatement questionStatement = connection.prepareStatement(insertQuestionQuery);
+      questionStatement.setString(SqlConstants.QUESTION_INSERT_TITLE_COLUMN, title);
+      questionStatement.setString(SqlConstants.QUESTION_INSERT_BODY_COLUMN, body);
+      questionStatement.setInt(SqlConstants.QUESTION_INSERT_ASKERID_COLUMN, asker_id);
+      questionStatement.executeUpdate();
+    } catch (SQLException exception) {
+      // If the connection or the query don't go through, we get the log of what happened.
+      Logger logger = Logger.getLogger(PostQuestionServlet.class.getName());
+      logger.log(Level.SEVERE, exception.getMessage(), exception);
+    }
+  }
+
+  /** 
+   * Gets the ID from the last question posted. Returns -1 on query failure.
+   */
+  private int getLatestQuestionId(Connection connection) {
+    int id = -1;
+    try {
+      String maxIdQuery = "SELECT MAX(id) FROM Question;";
+      PreparedStatement maxIdStatement = connection.prepareStatement(maxIdQuery);
+      ResultSet queryResult = maxIdStatement.executeQuery();
+      queryResult.next();
+      id = queryResult.getInt(SqlConstants.QUESTION_FETCH_MAXID_COLUMN);
+    } catch (SQLException exception) {
+      // If the connection or the query don't go through, we get the log of what happened.
+      Logger logger = Logger.getLogger(PostQuestionServlet.class.getName());
+      logger.log(Level.SEVERE, exception.getMessage(), exception);
+    }
+
+    return id;
+  }
+
+  /** 
+   * Makes the author of the recently added question a follower of said question.
+   * TODO(shaargtz): Move function to Utiliy class to be reused.
+   */
+  private void insertNewFollower(Connection connection, int asker_id) {
+    int latestQuestionId = getLatestQuestionId(connection);
+    try {
+      String insertFollowerQuery = "INSERT INTO QuestionFollower(question_id, follower_id) "
+          + "VALUES (?,?)";
+      PreparedStatement followerStatement = connection.prepareStatement(insertFollowerQuery);
+      followerStatement.setInt(SqlConstants.FOLLOWER_INSERT_QUESTIONID_COLUMN, latestQuestionId);
```

Comment:
nit: you only use latestQuestionId here, so you could instead just call getLatestQuestionId(connection) right here, rather than initializing it outside the try / catch.